src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -2196,6 +2196,21 @@ public FixableDiagnostic ProviderDeclarationViaImportKeywordIsDeprecated(Provide
                 warnInsteadOfError ? DiagnosticLevel.Warning : DiagnosticLevel.Error,
                 "BCP403",
                 $"The enclosing array expects elements of type \"{expectedType}\", but the array being spread contains elements of incompatible type \"{actualType}\".");
+
+            public ErrorDiagnostic ExtendsPathHasNotBeenSpecified() => new(
+                TextSpan,
+                "BCP404",
+                "The extends declaration is missing a bicepparam file path reference.");
+
+            public ErrorDiagnostic MoreThanOneExtendsDeclarationSpecified() => new(
+                TextSpan,
+                "BCP405",
+                "More than one 'extends' declaration are present");
+
+            public ErrorDiagnostic ExtendsNotSupported() => new(
+                TextSpan,
+                "BCP406",
+                "Extends keyword is not supported");
```

Comment:
I would refer to this consistently across all error messages, and also use double quotes: `$"The \"{LanguageConstants.ExtendsKeyword}\" keyword"`

Reply:
done 👍